sdk/trace/trace_test.go

```diff
@@ -482,130 +482,6 @@ func TestEventsOverLimit(t *testing.T) {
 	}
 }
 
-func TestAddLinks(t *testing.T) {
```

Comment:
Instead of removing this tests, lets change it to tests the `apitrace.LinkedTo()`. This would ensure that `Links` are covered by tests since they still exists while creating a Span. 

We would need to keep 2 out of these 3 tests. One for `LinkedTo` and other for `TestLinksOverLimit`.

Reply:
So out of 3 unit tests, I removed ```TestAddLinks``` as we completely removed ```AddLinks``` 
```TestLinks``` & ```TestLinksOverLimit``` needs validation changes as ```Link``` which I removed and ```LinkedTo```  which you requested to replace with ```Link``` are returning different values. I noticed ```LinkedTo``` is returning ```nil``` so should I change the unit tests to expect ```nil ```?
Also ```TestLinksOverLimit``` is expecting a ```DroppedLinkCount: 1``` which with ```LinkedTo``` is returning ```0```

As I am still a newbie with tracing couldn't figure this out. Please help me in getting started and in contributing to this project. :)

Reply:
Also is there any demo app explaining all ```links``` and other functionalities or a blog post to understand otel client capabilities and functionalities.

Reply:
rghetia has made some comments of what you should do.  Basically, `apitrace.LinkedTo` is a span start option that must be passed to the `startSpan`, this option informs that this span is _casually_  linked to another span.

There are some high level documentation on the opentelemetry-specs repository which can be a good start point for an [overview of distributed tracing](https://github.com/open-telemetry/opentelemetry-specification/blob/master/specification/overview.md).